app/controllers/contents_controller.rb

```diff
@@ -0,0 +1,34 @@
+class ContentsController < ApplicationController
+  def index
+    @contents = Form.find(params[:id]).contents.includes(:submissions)
```

Comment:
How does your endpoints look like? `forms/:form_id/contents`? 

If so, you should do

```ruby
@contents = Form.find(params[:form_id]).contents.includes(:submissions)
```